packages/desktop-client/src/components/accounts/Header.jsx

```diff
@@ -501,36 +506,39 @@ function AccountMenu({
       items={[
         isSorted && {
           name: 'remove-sorting',
-          text: 'Remove all sorting',
+          text: t('Remove all sorting'),
         },
         canShowBalances && {
           name: 'toggle-balance',
-          text: (showBalances ? 'Hide' : 'Show') + ' running balance',
+          text: (showBalances ? t('Hide') : t('Show')) + t(' running balance'),
```

Comment:
This is a good example for strings that should be translated as a whole. 

A translator who will only see `" running balance"` won't be able to make an informed decision about the translation without the "Show" or "Hide" context.

Reply:
Agreed (for this and the other occurrences below as well)